build

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# Locally builds the docker container and runs tests.
+#
+# The test sets to run (gpu, cpu, tpu) can be specified as arguments.  By
+# default all tests are executed.
+
+PROJECT_ID="$(gcloud config get-value project)"
+if [ -z "${PROJECT_ID}" ]; then
+  echo "You must set a project ID with `gcloud config set`"
+  exit 1
+fi
+
+MODEL="gs://deepvariant/models/DeepVariant/0.7.2/DeepVariant-inception_v3-0.7.2+data-wgs_standard"
+IMAGE="gcr.io/${PROJECT_ID}/gcp-deepvariant-docker:HEAD"
+
+docker build -t "${IMAGE}" .
+docker push "${IMAGE}"
+
+for profile in ${@:-cpu gpu tpu}; do
```

Comment:
For TPU, one need to configure its project such that cloud-tpu has read access to the GCS bucket.

Please add a comment on top with the following link

https://cloud.google.com/tpu/docs/storage-buckets#storage_access

Reply:
This was another thing I hit that made me feel the TPU runner wasn't quite ready for user consumption.  This error wasn't surfaced in a very friendly way - I had to dig into the GKE logs to find out what the problem was.

I can add a comment to the build file but it seems unlikely that anyone will see it. I think the better path is to check the IAM policy and issue a warning if it's not set properly (it's what our tooling should do, too).  I'll look at doing that.

Reply:
This is a requirements for using cloud TPU explained in the cloud page. Automatically detecting it and issuing warning is a good feature. We also have a bug for better GKE error reporting, maybe we should convert all the outstanding bugs into Github issues.

Reply:
Yes any outstanding bugs should be migrated to Github issues: please assign all the outstanding bugs to me and I'll migrate them.